astroquery/jplhorizons/core.py

```diff
@@ -149,6 +149,7 @@ def ephemerides_async(self, airmass_lessthan=99,
                           refsystem='J2000',
                           closest_apparition=False, no_fragments=False,
                           quantities=conf.eph_quantities,
+                          optional_settings=None,
```

Comment:
This introduces an API break, so the new kwarg has to come to the end of kwargs, or make kwargs keyword only (I would prefer that latter)